project.clj

```diff
@@ -7,7 +7,11 @@
             :year 2017
             :key "gpl-3.0"}
 
-  :dependencies [[org.clojure/clojure "1.9.0"]
+  ;; fix for jdk 10
+  :managed-dependencies [[org.clojure/core.rrb-vector "0.0.14"]
```

Comment:
what are those needed for?

Reply:
Please share link with suggested fix. Was it failing without it?

Reply:
was link you also mentioned best source for upgrade